conda_env/cli/main_remove.py

```diff
@@ -32,7 +32,7 @@ def configure_parser(sub_parsers):
     )
 
     add_parser_prefix(p)
-    add_parser_experimental_solver(p)
+    add_parser_solver(p)
```

Comment:
Should this one also have an alias for now?

Reply:
I don't think so. This has not been in conda for that long, so I really don't think there are other projects out there depending on it. Plus, this would be a weird thing for other people to depend on.